app/models/like.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+class Like extends AppModel
+{
+    public static function count($comment_id)
+    {
+        $db = DB::conn();
+        return (int) $db->value('SELECT COUNT(*) FROM likes WHERE comment_id = ?', array($comment_id));
+    }
+
+    public function comment()
+    {
+        $db = DB::conn();
+        $like_id = (int) $db->value('SELECT id FROM likes
+        WHERE user_id = ? AND comment_id = ?', array($this->user_id, $this->comment_id));
+        if ($this->type === 'like' && $like_id === 0) {
+            self::addLike($this->user_id, $this->comment_id);
```

Comment:
Hello sir, deleted this part as the result of refactoring the next comment. Thanks! :bow: